modules/navier_stokes/test/tests/finite_volume/ins/turbulence/channel/linear-segregated-transient/channel_ERCOFTAC.i

```diff
@@ -0,0 +1,440 @@
+    boundary = 'left'
+    variable = vel_y
+    functor = '0.0'
+  []
+  [walls-u]
+    type = LinearFVAdvectionDiffusionFunctorDirichletBC
+    boundary = 'top bottom'
+    variable = vel_x
+    functor = 0.0
+  []
+  [walls-v]
+    type = LinearFVAdvectionDiffusionFunctorDirichletBC
+    boundary = 'top bottom'
+    variable = vel_y
+    functor = 0.0
+  []
+  [outlet_u]
+    type = LinearFVAdvectionDiffusionOutflowBC
+    boundary = 'right'
+    variable = vel_x
+    use_two_term_expansion = false
+  []
+  [outlet_v]
+    type = LinearFVAdvectionDiffusionOutflowBC
+    boundary = 'right'
+    variable = vel_y
+    use_two_term_expansion = false
+  []
+  [outlet_p]
+    type = LinearFVAdvectionDiffusionFunctorDirichletBC
+    boundary = 'right'
+    variable = pressure
+    functor = 0.0
+  []
+
+  [inlet_TKE]
+    type = LinearFVAdvectionDiffusionFunctorDirichletBC
+    boundary = 'left'
+    variable = TKE
+    functor = '${k_init}'
+  []
+  [outlet_TKE]
+    type = LinearFVAdvectionDiffusionOutflowBC
+    boundary = 'right'
+    variable = TKE
+    use_two_term_expansion = false
+  []
+  [inlet_TKED]
+    type = LinearFVAdvectionDiffusionFunctorDirichletBC
+    boundary = 'left'
+    variable = TKED
+    functor = '${eps_init}'
+  []
+  [outlet_TKED]
+    type = LinearFVAdvectionDiffusionOutflowBC
+    boundary = 'right'
+    variable = TKED
+    use_two_term_expansion = false
+  []
+[]
+
+[FVBCs]
+  [walls_mu_t]
+    type = INSFVTurbulentViscosityWallFunction
+    boundary = 'bottom top'
+    variable = mu_t
+    u = vel_x
+    v = vel_y
+    rho = ${rho}
+    mu = ${mu}
+    mu_t = 'mu_t'
+    k = TKE
+    wall_treatment = ${wall_treatment}
+  []
+[]
+
+[AuxVariables]
+  [mu_t]
+    type = MooseVariableFVReal
+    initial_condition = '${fparse rho * C_mu * ${k_init}^2 / eps_init}'
+    two_term_boundary_expansion = false
+  []
+  [yplus]
+    type = MooseVariableFVReal
+    two_term_boundary_expansion = false
+  []
+  [mu_eff]
+    type = MooseVariableFVReal
+    initial_condition = '${fparse rho * C_mu * ${k_init}^2 / eps_init}'
+    two_term_boundary_expansion = false
+  []
+[]
+
+[AuxKernels]
+  [compute_mu_t]
+    type = kEpsilonViscosityAux
+    variable = mu_t
+    C_mu = ${C_mu}
+    k = TKE
+    epsilon = TKED
+    mu = ${mu}
+    rho = ${rho}
+    u = vel_x
+    v = vel_y
+    bulk_wall_treatment = ${bulk_wall_treatment}
+    walls = ${walls}
+    wall_treatment = ${wall_treatment}
+    execute_on = 'NONLINEAR'
+    mu_t_ratio_max = 1e20
+  []
+  [compute_y_plus]
+    type = RANSYPlusAux
+    variable = yplus
+    k = TKE
+    mu = ${mu}
+    rho = ${rho}
+    u = vel_x
+    v = vel_y
+    walls = ${walls}
+    wall_treatment = ${wall_treatment}
+    execute_on = 'NONLINEAR'
+  []
+  [compute_mu_eff]
+    type = ParsedAux
+    variable = 'mu_eff'
+    coupled_variables = 'mu_t'
+    expression = 'mu_t + ${mu}'
+    execute_on = 'NONLINEAR'
+  []
+[]
+
+[Executioner]
+  type = PIMPLE
```

Comment:
I think we could put this and the steady version ot the same folder and reuse most of the input files. You can use the `!include mesh.i` syntax etc.

Reply:
I would like to have complete working tests based on the feedback that I have received from our users. The include requires people accessing two files instead of one, which complicates the understanding. Thank you

Reply:
Fine.